types/secure-random-password/index.d.ts

```diff
@@ -1,20 +1,28 @@
-// Type definitions for secure-random-password 0.1
+// Type definitions for secure-random-password 0.2
 // Project: https://github.com/rackerlabs/secure-random-password
 // Definitions by: Bjørnar Snoksrud <https://github.com/bjornars>
+//                 GaspardFRDev <https://github.com/GaspardFRDev>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export const lower: string;
-export const upper: string;
 export const consonants: string;
-export const vowels: string;
+export const copyableSymbols: string;
```

Comment:
Personally, I find the alphabetized list less clear.

Reply:
Ok, I can simply add space between each categorie then ?
```
export const lower: string;
export const upper: string;

export const consonants: string;
export const vowels: string;
export const digits: string;

export const symbols: string;
export const copyableSymbols: string;
export const fullSymbols: string;

```

Reply:
Works for me.